src/utils/endUserMessages.js

```diff
@@ -3,18 +3,18 @@ export default {
   DELETING_PROJECT: 'Deleting project...',
   DELETING_SAMPLE: 'Deleting sample...',
   EMPTY_CLUSTER_NOT_CREATED: 'Cannot create a cluster that would be empty.',
-  ERROR_DELETING_PROJECT: 'The server returned an error while deleting your project',
-  ERROR_DELETING_SAMPLES: 'The server returned an error while deleting samples from your project',
-  ERROR_FETCHING_CELL_SETS: 'The server returned an error while getting the list of cell sets',
-  ERROR_FETCHING_EXPERIMENTS: 'The server returned an error while getting the list of experiments',
-  ERROR_FETCHING_PLOT_CONFIG: 'The server returned an error while getting plot settings',
-  ERROR_FETCHING_PROCESSING: 'The server returned an error while getting filter settings',
-  ERROR_FETCHING_PROJECTS: 'The server returned an error while getting the list of projects',
-  ERROR_FETCHING_SAMPLES: 'The server returned an error while getting the samples',
-  ERROR_FETCHING_STATUS: 'The server returned an error while getting the status of pending jobs',
-  ERROR_NOT_SIGNED_IN: 'You need to sign in. Click the button on the top right.',
-  ERROR_SAVING: 'The server returned an error while saving your data',
-  ERROR_STARTING_PIPLELINE: 'The server returned and error while trying to start working on your data',
+  ERROR_DELETING_PROJECT: 'We couldn\'t delete your project.',
+  ERROR_DELETING_SAMPLES: 'We couldn\'t delete samples from your project.',
+  ERROR_FETCHING_CELL_SETS: 'We couldn\'t get the list of cell sets.',
+  ERROR_FETCHING_EXPERIMENTS: 'We couldn\'t get the list of experiments.',
+  ERROR_FETCHING_PLOT_CONFIG: 'We couldn\'t get the plot settings.',
+  ERROR_FETCHING_PROCESSING: 'We couldn\'t get the filter settings.',
+  ERROR_FETCHING_PROJECTS: 'We couldn\'t get the list of projects.',
+  ERROR_FETCHING_SAMPLES: 'We couln\'t get your samples.',
+  ERROR_FETCHING_STATUS: 'We couldn\'t get the status of pending jobs.',
+  ERROR_NOT_SIGNED_IN: 'You need to sign in first. Click Sign in on the top right to begin.',
+  ERROR_SAVING: 'We couldn\'t save your data.',
+  ERROR_STARTING_PIPLELINE: 'We couldn\'t get our server to start working on your data.',
```

Comment:
Based on the changes suggested I think this could be "We couldn\'t start working on your data.", other than that looks good